### PR TITLE
Issue 4 home page

### PR DESCRIPTION
**Pull request by Julie-Salazar**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**
fixed import warnings and cleaned text in home page
### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

### CODE REVIEW

**Comment by MouseAndKeyboard** [comment]
oh no julie what have you done

**Comment by jsun1590** [comment]
Julie....

File: ~
Comment (cjnbennett) [inline ~:1]: I assume this file probably wasn't meant to be committed

**Comment by jsun1590** [comment]
@Julie-Salazar Please change your private keys ASAP...

File: CFC
Comment (cjnbennett) [inline CFC:1]: You'll probably want to revoke this ssh key now

**Review by cjnbennett — CHANGES_REQUESTED** [review]
The changes look good but I think there are a few stray files that have been committed as well (`CFC`, `CFC.pub`, `~`), including your SSH keypair
It looks like there's also a merge conflict to resolve. I'll can help you out with it tomorrow if you like

**Review by cjnbennett — APPROVED** [review]

**Pull request merged**